homeassistant/components/remote/harmony.py

```diff
@@ -106,6 +106,9 @@
     try:
         device = HarmonyRemote(
             name, address, port, activity, harmony_conf_file, delay_secs)
+        if not await device.connect():
+            raise PlatformNotReady
+
         DEVICES.append(device)
```

Comment:
A comment for a future PR:
Devices should not be stored in a constant but in hass.data.